server/etcdserver/server_test.go

```diff
@@ -514,8 +514,14 @@ func TestApplyRequestOnAdminMemberAttributes(t *testing.T) {
 }
 
 func TestApplyConfChangeError(t *testing.T) {
```

Comment:
Don't understand the test changes. Please add a test that will confirm the change you introduced. Now we not only validate config change based on v2 state, but also from v3 state. I expect we should add a test that confirms that v3 is validated.

Reply:
TestClusterValidateConfigurationChange and TestApplyConfChangeError now set backend object for the RaftCluster. As a result, these tests will exercise the v3 validation from here - https://github.com/etcd-io/etcd/pull/16084/files#diff-09b9c924aaa34d5ad5abc7e40ed10a2dcc5257c62fda537a8671a7f206d03e6aR317